tests/kernel/stack/stack_api/prj.conf

```diff
@@ -2,3 +2,5 @@ CONFIG_ZTEST=y
 CONFIG_IRQ_OFFLOAD=y
 CONFIG_USERSPACE=y
 CONFIG_DYNAMIC_OBJECTS=y
+CONFIG_OBJECT_TRACING=y
```

Comment:
what does CONFIG_OBJECT_TRACING have to do with this test?

Reply:
@ajaykish if you are doing this to increase coverage, then you should do this in a separate patch and add this to some test project that validates object_tracing, not here.